lib/xandra/connection_error.ex

```diff
@@ -92,6 +92,13 @@ defmodule Xandra.ConnectionError do
     "not connected to any of the nodes"
   end
 
+  defp format_reason({:cluster, :pool_closed}) do
+    """
+    the Xandra pool is closed, probably because the Cassandra connection dropped right after
```

Comment:
This is also a Scylla driver technically 😄 

```suggestion
    the Xandra pool is closed, probably because the network connection dropped right after
```